### PR TITLE
Add filter to allow customizing who can flush the cache

### PR DESCRIPTION
**Pull request by pmgarman**
Couldn't find a CONTRIBUTING.md (i may be blind) so took a best guess. Also just assumed 2.6 but who knows?!

I think this could easily be a thing that gets enhanced deeper for multisite to allow individual site owners to flush their individual sites cache or something along those lines... but this probably serves my needs well enough :D

### CODE REVIEW

**Comment by pmgarman** [comment]
This was the code i used to test that this worked for the author role:
```php
add_filter( 'roc_manage_redis_capability', function( $cap ) {
	return 'publish_posts';
});
```

**Comment by pmgarman** [comment]
Resolved conflict in changelog

**Comment by tillkruss** [comment]
Should we also make this a constant for easier access?

**Comment by pmgarman** [comment]
You would only be able to define your default as a constant though right? Since the constant can't change after being filtered?

**Comment by tillkruss** [comment]
> You would only be able to define your default as a constant though right? Since the constant can't change after being filtered?

Yeah, it would be set once in the `wp-config.php` and should be opt-in, not used by default.

**Comment by pmgarman** [comment]
Oh I see what you mean, inverse of what I was thinking. It's monday...

I could see that being handy too yeah. I can update my PR and fix those codestyle complaints.

Any preference on your constant name?

**Comment by pmgarman** [comment]
Nevermind those errors are yours :P enjoy!

**Comment by pmgarman** [comment]
Opted for `WP_REDIS_MANAGER_CAPABILITY` until @tillkruss says otherwise :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Added `roc_manage_redis_capability` filter and `WP_REDIS_MANAGER_CAPABILITY` constant
```
Comment (tillkruss) [inline CHANGELOG.md:5]: @pmgarman: Could you add these to the README as well?

**Review by tillkruss — CHANGES_REQUESTED** [review]

**Comment by pmgarman** [comment]
@tillkruss try this on for size

**Pull request merged**